### ISSUE TITLE
Why does SSD appear to be less reliant on its feature extractor’s classification accuracy?

### ISSUE THREAD

**Issue by Sundrops** [weiliu89/caffe#364]
Hi Wei Liu,
Firstly,thanks for sharing the code! I find the fact that SSD appears to be less reliant on its feature extractor’s classification accuracy through this plot.But why? Thanks a lot!
![ssd](https://cloud.githubusercontent.com/assets/14829222/21581887/5ad21df6-d07e-11e6-8a5b-8f758394c1dc.png)
Huang J, Rathod V, Sun C, et al. Speed/accuracy trade-offs for modern convolutional object detectors[J]. 2016. [https://arxiv.org/abs/1611.10012](url)

**Comment by weiliu89** [comment]
@Sundrops First of all, I think the paper doesn't exactly reproduce SSD in TensorFlow. You can check our latest number. A colleague of mine has successfully trained a ResNet version of SSD, which has about 30ish mAP on COCO. I would say, 'devil is in the details'. The results doesn't favor R-FCN neither, but R-FCN seem to achieve better results on COCO as reported in this paper.

Different networks have different characteristics. I think you have to search a bit to find the optimal results for each base network and detector framework. For example, I think Faster R-CNN and R-FCN can do better on Inception V2.

**Comment by Sundrops** [comment]
Thanks a lot! Devil is in the details. I will have a try.
